moped-editor/src/views/projects/projectView/ProjectWorkActivity/ProjectWorkActivityTable.js

```diff
@@ -118,8 +120,7 @@ const useColumns = ({ deleteInProgress, onDeleteActivity, setEditActivity }) =>
         field: "updated_at",
         width: 150,
         defaultVisible: true,
-        type: "date",
-        valueGetter: (field) => (field ? new Date(field) : null),
+        valueGetter: (field) => (field ? formatRelativeDate(field) : null),
```

Comment:
I think this is great for faster comprehension of when a record was updated, and we can now compare when records were updated within the same day which we couldn't do before. 🚀

Beyond this PR, this made me think even more about how we display timestamps elsewhere. We show the same level of info for status updates in the Project Summary page, but we show both this human-readable format and the actual date and time in the activity log for example.

<img width="163" alt="Screenshot 2024-09-04 at 10 50 12 AM" src="https://github.com/user-attachments/assets/0e802b3d-1321-4075-8e4a-3bcfb0c11490">

Looking for feedback because I think a future issue to make a reusable DataGrid cell component for timestamps could be interesting. Another place would be the **Modified** column of the Project List view for example.

Reply:
I was just looking at this, and came to the code looking for a place to share a very similar opinion to what you have here Mike. I am all-in on the relative date, but not at the expense of the absolute date. I would fully support enhancing this component to render the date as you have it in your screenshot, or at the very least, showing the absolute date on-mouseover of the relative date, perhaps with some light text decoration in the form of a dashed underline.

Reply:
that sounds like a cool idea to me, Mateo! thanks for the discussion, y'all. There are other cells that use `renderCell` instead of `valueGetter` if you want to see some examples of cells using components. And the screenshot that I shared comes from the `ProjectActivityLog` component.

Reply:
there are some differences with renderCell vs valueGetter and valueFormatter with how DataGrid uses them to sort, just throwing that out there
https://mui.com/x/react-data-grid/column-definition/#value-getter

We'd have to include a sortComparator in that case.

Reply:
this all sounds great—i'm glad y'all are going to get this standardized across the app. fwiw, Github populates the full timestamp into the `title` attribute on their timestamp elements so that it appears on mouse-over via built-in browser behavior. it would be easy enough for us to do that, but i don't find it intuitive at all and i doubt our users would know to look for that behavior:

screenshot of this hover effect:
<img width="470" alt="Screenshot 2024-09-05 at 11 19 32 AM" src="https://github.com/user-attachments/assets/eb2a6b20-672e-404c-a7a0-0f34c4e28e31">